src/info/info.html

```diff
@@ -0,0 +1,19 @@
+<template>
+
+    <div class="row">
+        <template repeat.for="column of columns">
+            <div class="col-sm-6 text-weight-bold">${column.label}</div>
+            <div class="col-sm-6">${column.value}</div>
+        </template>
+    </div>
+    <div class="row">
+        <div class="col-sm-6 text-weight-bold">Channels:</div>
+        <div class="col-sm-6">
+            <template repeat.for="channel of image_info.channels">
+                <span if.bind="!$last">${channel.label},</span>
```

Comment:
This is most definitely a way of doing it in aurelia, another more concise way without using if.bind is: 

```
<span>${channel.label + !$last ? ',' : ''}</span> 
```

I tend to use it because the if creates comment placeholders in the html and the html template itself is a bit leaner, at the expense of having more complicated conditions I agree. So I usually use the if.bind method for large sections of "more different" html, e.g. the rois table where I have more different looking rows with event hooks that exist in one type of row but not the other. in such cases I do not know how else to handle such situations.